inttest/Makefile

```diff
@@ -10,6 +10,12 @@ bins = bin/sonobuoy
 
 include ../embedded-bins/Makefile.variables
 
+ifeq ($(ARCH),amd64)
+etcd_arch = amd64
+else
+etcd_arch = arm64
+endif
```

Comment:
why not `etcd_arch = $(ARCH)`?

Reply:
I didn't know that value of `$(ARCH)` on armv7 is "arm64". It seems that `etcd_arch` is not necessary, so I will remove this variable and use `$(ARCH)` as a build arg.

Reply:
I can see in logs that `$(ARCH)` on armv7 is "arch", so I have to revert my last commit.

Reply:
`etcd_arch` must be equal to "amd64" or "arm64", because etcd has no release for armv7 architecture.